.github/workflows/changed.yaml

```diff
@@ -0,0 +1,106 @@
+name: Deploy references.toml
+on:
+  push:
+#    branches:
+#      - main
+#    paths:
+#      - 'references.py'
+
+jobs:
+  make_matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set_matrix.outputs.matrix }}
+    env:
+      REFERENCES_PREFIX: gs://cpg-reference-test
```

Comment:
Needs to be changed to the actual dataset path?

Reply:
Will change that to reading secrets created by infra pulumi